setup.cfg

```diff
@@ -5,7 +5,7 @@ long_description = 'drizzle tools: combines astronomical images, including '
                    'modeling distortion, removing cosmic rays, and generally '
```

Comment:
It may be safe to remove `setup.cfg`, however, I would like to take a couple of releases to deprecate the use of `setup.py` for installing this package.  After all, even the JWST package still retains both of these configuration files and it will allow users to learn how to use newer syntax while still allowing them to use the current 'python setup.py' syntax in the meantime.  Forcing the situation at this time would only insight more help calls, no matter how clearly documented it may be.

Reply:
Sure thing! In that case, we can remove all of the parameters from `setup()` to make it a shim for legacy usage of `python setut.py install` as described at https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html:
```python
from setuptools import setup

setup()
```
Setuptools will retrieve values from `pyproject.toml` and insert them into the `setup()` call.

Reply:
In addition, testing the installation process using the command `pip install -e .` without either of those files results in the following error:

> ERROR: File "setup.py" or "setup.cfg" not found. Directory cannot be installed in editable mode: D:\Drive_D\cygwin\home\Warren Hack\github\drizzlepac
> (A "pyproject.toml" file was found, but editable mode currently requires a setuptools-based build.)

As a result, I will make sure all the info from `setup.cfg` has been included in `pyproject.html` and then remove `setup.cfg`.

Reply:
Oh, it is a bit funky to build an editable installation with only `pyproject.toml` with C extensions, so a `setup.py` shim should be sufficient in solving that issue.